projects/step-frontend/src/lib/modules/report-nodes/components/call-function-report-node/call-function-report-node.component.ts

```diff
@@ -17,13 +18,37 @@ export class CallFunctionReportNodeComponent implements OnChanges {
 
   hideMeasures: boolean = false;
 
+  execution: Execution;
+
+  constructor(private executionService: ExecutionStateService) {
```

Comment:
If execution service doesn't use anywhere else except constructor, `private` can be omitted